From 1cd691608085920eb1421efda52dc58ee25bd28c Mon Sep 17 00:00:00 2001
From: Vaibhav Gumashta <vgumashta@hortonworks.com>
Date: Fri, 1 Jul 2016 18:48:06 -0700
Subject: [PATCH 0710/1164] CDH-42598 : HIVE-14153: Beeline: beeline history
 doesn't work on Hive2 (Vaibhav Gumashta reveiwed
 by Thejas Nair)

Change-Id: Ib227edd5c5cc6f1b8b5ca34f37a429a33d2cb9ce
---
 bin/beeline |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/bin/beeline b/bin/beeline
index 4360ae0..c23ab22 100644
--- a/bin/beeline
+++ b/bin/beeline
@@ -18,11 +18,6 @@
 bin=`dirname "$0"`
 bin=`cd "$bin"; pwd`
 
-# If process is backgrounded, don't change terminal settings
-if [[ ! $(ps -o stat= -p $$) == *+ ]]; then
-  export HADOOP_CLIENT_OPTS="$HADOOP_CLIENT_OPTS -Djline.terminal=jline.UnsupportedTerminal"
-fi
-
 # Set Hadoop User classpath to true so that httpclient jars are taken from
 # hive lib instead of hadoop lib.
 export HADOOP_USER_CLASSPATH_FIRST=true
-- 
1.7.9.5

